src/app/containers/StatsPage/components/AmmBalance.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { backendUrl, currentChainId } from '../../../../utils/classifiers';
+import { SkeletonRow } from '../../../components/Skeleton/SkeletonRow';
+import { AmmBalanceRow } from '../types';
+import { formatNumber } from '../utils';
+
+export function AmmBalance() {
+  const assets = ['DOC', 'USDT', 'BPRO'];
```

Comment:
Change these to use Asset.DOC / Asset.USDT / Asset.BPRO to prevent issues if one asset will need to be renamed, removed, etc.